ios/FluentUI/Pill Button Bar/PillButton.swift

```diff
@@ -151,7 +151,7 @@ open class PillButton: UIButton, TokenizedControlInternal {
             }
```

Comment:
You might also need to add a title transformer in the iOS 15 section.
```
let titleTransformer = UIConfigurationTextAttributesTransformer { incoming in
                var outgoing = incoming
                outgoing.font = UIFont.fluent(self.tokenSet[.titleFont].fontInfo)
                return outgoing
            }
```
The Button has an example of this.